### DISCUSSION TITLE
Bun v1.0.11

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#6989] [Announcements]
To install Bun v1.0.11:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade to Bun v1.0.11:
```bash
bun upgrade
```

**[Read Bun v1.0.11's release notes on Bun's blog](https://bun.sh/blog/bun-v1.0.11)**

## What's Changed
* Fix for #6951 by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6952
* fix(handshake) by @cirospaciari in https://github.com/oven-sh/bun/pull/6960
* improve/fix on jsonwebtoken tests by @cirospaciari in https://github.com/oven-sh/bun/pull/6958
* Fixes #6973 by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6975
* Add missing `raw` method to node-fetch polyfill by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6976
* Fixes #6200 by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6983
* fix semver edge cases and buffer used for prerelease comparisons by @dylan-conway in https://github.com/oven-sh/bun/pull/6981
* Update edgedb.md by @MuhibAhmed in https://github.com/oven-sh/bun/pull/6985

## New Contributors
* @MuhibAhmed made their first contribution in https://github.com/oven-sh/bun/pull/6985

**Full Changelog**: https://github.com/oven-sh/bun/compare/bun-v1.0.10...bun-v1.0.11

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.0.11'>Bun v1.0.11</a>.</em>

**Comment by oenu** [comment]
Great to see progress being made!
